traveller/modules/schedule/view.py

```diff
@@ -21,6 +26,12 @@
 globals()[mhelp.blueprint_str] = mhelp.blueprint
 module_blueprint = globals()[mhelp.blueprint_str]
 
+
+def _date_difference(start_date, end_date):
+    if end_date < start_date:
+        return True
```

Comment:
That function makes the code less readable. Just put the check in the places where you call it atm...